XpressAI/xircuits
# Redesigning the sidebar

Description:
- Redesigning the side bar without using the use of material UI design.
- Added CSS to the sidebar and input.
- Added force update to get the components for every 15 seconds.
- Removed autocomplete. Since it was feature comes with material UI.
- Added group components to make code reusable.

To test:
- Do test if the search function is working correctly.
- Do test if the dragging and dropping is working correctly.
- Do check if the group components are working correctly, Whatever is in the "XAI_LEARNING" will be just "LEARNING".

![image](https://user-images.githubusercontent.com/23378929/137472303-a39fb02a-946c-4436-a9a0-ee52cceff1c6.png)
